dist/profile/templates/confluence/vhost.conf

```
@@ -179,6 +179,8 @@ RewriteRule "^/display/JENKINS/ArtifactDeployer\+Plugin$" "https://plugins.jenki
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Artifactory\+Plugin$" "https://plugins.jenkins.io/artifactory" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Artifactory\+-\+Working\+With\+Pipeline\+Jobs\+in\+Jenkins#" "https://plugins.jenkins.io/workflow-aggregator" [NC,L,QSA,R=301]
```

The [source page](https://wiki.jenkins.io/display/JENKINS/Artifactory+-+Working+With+Pipeline+Jobs+in+Jenkins#) says:

> This page has been moved. Click [here](https://www.jfrog.com/confluence/display/JFROG/Working+With+Pipeline+Jobs+in+Jenkins) to get to the new page.

This redirect should point to that location, not to the workflow aggregator.

```suggestion
RewriteRule "^/display/JENKINS/Artifactory\+-\+Working\+With\+Pipeline\+Jobs\+in\+Jenkins#" "https://www.jfrog.com/confluence/display/JFROG/Working+With+Pipeline+Jobs+in+Jenkins" [NC,L,QSA,R=301]
```

---

ohh yes, sorry.